netam/models.py

```diff
@@ -1000,9 +1028,41 @@ def forward(
         Inputs should be in unflattened form, with codons represented in shape (4, 4,
         4).
         """
+        result = self.apply_multihit_correction(
+            parent_codon_idxs, uncorrected_codon_probs
+        )
+        # clamp only above to avoid summing a bunch of small fake values when
```

Comment:
Could we pull this out into a function with a name and a slightly clearer docstring? This is a little on the opaque side.